crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -444,6 +444,8 @@ en:
       identify: 'Identify'
     form:
       allocate: 'Allocate'
+      confirm_allocate: 'WARNING! Allocating the node will wipe all the disks attached to the node including the shared disks. Proceed?'
+      allocate_tooltip: 'Cleanup and Allocate node with the intended role'
```

Comment:
I don't think you should mention "intended role" here as it might not be set. Also, what does "cleanup" mean?